pkg/assembler/graphql/resolvers/artifact.resolvers.go

```diff
@@ -12,13 +12,24 @@ import (
 )
 
 // IngestArtifact is the resolver for the ingestArtifact field.
-func (r *mutationResolver) IngestArtifact(ctx context.Context, artifact *model.ArtifactInputSpec) (*model.Artifact, error) {
-	return r.Backend.IngestArtifact(ctx, artifact)
+func (r *mutationResolver) IngestArtifact(ctx context.Context, artifact *model.ArtifactInputSpec) (string, error) {
+	ingestedArtifact, err := r.Backend.IngestArtifact(ctx, artifact)
+	if err != nil {
+		return "", err
+	}
+	return ingestedArtifact.ID, err
 }
 
 // IngestArtifacts is the resolver for the ingestArtifacts field.
-func (r *mutationResolver) IngestArtifacts(ctx context.Context, artifacts []*model.ArtifactInputSpec) ([]*model.Artifact, error) {
-	return r.Backend.IngestArtifacts(ctx, artifacts)
+func (r *mutationResolver) IngestArtifacts(ctx context.Context, artifacts []*model.ArtifactInputSpec) ([]string, error) {
+	ingestedArtifacts, err := r.Backend.IngestArtifacts(ctx, artifacts)
```

Comment:
Blocker comment to ensure that this does not auto merge. Will need to update the backends.